addons/web/static/src/search/control_panel/control_panel.xml

```diff
@@ -16,42 +16,33 @@
                 </div>
             </div>
             <div t-if="display['bottom']" class="o_cp_bottom">
-                <t t-if="display['bottom-content']">
```

Comment:
there are simpler alternatives?

Reply:
Simple, not necessarily simpler (from the dashboard pov)